simulator/func_sim/alu.h

```diff
@@ -225,6 +225,20 @@ struct ALU
     // Bit manipulations
     template<typename I, typename T> static void pack( I* instr)  { instr->v_dst = (instr->v_src1 & (bitmask<T>(half_bitwidth<T>))) | (instr->v_src2 << (half_bitwidth<T>)); }
 
+    // Bit Extract/Deposit
+    template<typename I> static
+    void bext(I* instr)
+    {
+        instr -> v_dst = 0;
```

Comment:
Please follow our code conventions regarding spaces between variables and statements.